lib/screens/alerts/alert.ex

```diff
@@ -154,6 +154,9 @@ defmodule Screens.Alerts.Alert do
   @type result :: {:ok, [t()]} | :error
   @type fetch :: (options() -> result())
 
+  # Internally used type for distinguishing different station closures we handle differently
+  @type closure_type :: :partial_closure | :full_station_closure | :partial_closure_multiple_stops
```

Comment:
You can require/enforce that this is only used within this module by making it a `@typep`. Though since this seems to only be used in a single place (the return type of `station_closure_type`), we could just as well inline it.

Reply:
had it as an inline initially, and think I'm just gonna move it back to that since it's only used there and quicker to reference then